src/Color.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Atk4\Chart;
+
+/**
+ * Color generator.
+ *
+ * @todo Replace this with something which would generate infinite colors.
+ * Like https://medium.com/code-nebula/automatically-generate-chart-colors-with-chart-js-d3s-color-scales-f62e282b2b41 or something else
+ */
+class Color
```

Comment:
I would remove `Color generator.` comment above the class and rename to `ColorGenerator`.

Reply:
will do